src/main/scala/com/gd/twitterstreamingtToDatalake/SparkSessionWrapper.scala

```diff
@@ -0,0 +1,14 @@
+package com.gd.twitterstreamingtToDatalake
+
+import org.apache.spark.SparkConf
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+
+trait SparkSessionWrapper {
+
+  //Set the Spark Streaming Context
```

Comment:
I don't think that the comment is useful, bacause: 
1) the code below is self-descriptive enough
2) there is no not evident business logic or operational peculiarity to clarify.

Please read chapter #4 on comments from "Clean code" book  (reference in the reading list)

Reply:
Removed !